src/ngraph/op/sum.hpp

```diff
@@ -72,34 +72,39 @@ namespace ngraph
         /// | ----------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
         /// | \f$N[\textit{delete}(A,d_1,\dots,d_n)]\f$ | The tensor \f$T\f$, where \f$T\f$ is the input tensor with the `reduction_axes` \f$A\f$ eliminated by summation. |
         // clang-format off
-        class Sum : public util::ArithmeticReduction
+        namespace v0
         {
-        public:
-            NGRAPH_API
-            static constexpr NodeTypeInfo type_info{"Sum", 0};
-            const NodeTypeInfo& get_type_info() const override { return type_info; }
-            /// \brief Constructs a summation operation.
-            Sum() = default;
-            /// \brief Constructs a summation operation.
-            ///
-            /// \param arg The tensor to be summed.
-            /// \param reduction_axes The axis positions (0-based) to be eliminated.
-            Sum(const Output<Node>& arg, const AxisSet& reduction_axes);
-            /// \brief Constructs a summation operation.
-            ///
-            /// \param arg The tensor to be summed.
-            /// \param reduction_axes The axis positions (0-based) to be eliminated.
-            Sum(const Output<Node>& arg, const Output<Node>& reduction_axes);
+            class Sum : public util::ArithmeticReduction
```

Comment:
Docstring for the class should be moved down as well.

Reply:
Done